app.rb

```diff
@@ -0,0 +1,65 @@
+require './student'
+require './teacher'
+require './book'
+require './classroom'
+
+class App
+  attr_reader :books, :classrooms, :students, :teachers
+
+  def initialize
+    @students = []
+    @books = []
+    @teachers = []
+    @rentals = []
+    @classrooms = []
+  end
+
+  def list_people
+    @students.each { |s| puts "[Student] ID: #{s.id}, Name: #{s.name}" }
+    @teachers.each { |t| puts "[Teacher] ID: #{t.id}, Name: #{t.name}" }
+  end
+
+  def list_books
+    @books.each { |b| puts "[Book] Title: #{b.title}, Author: #{b.author}" }
+  end
+
+  def list_classrooms
+    @classrooms.each { |c| puts "[Classroom] Label: #{c.label}" }
+  end
+
+  def create_student(age, name, classroom)
+    student = Student.new(age, name, classroom)
+    @students << student
+    student
+  end
+
+  def create_teacher(age, name, specialization)
+    teacher = Teacher.new(age, name, specialization)
+    @teachers << teacher
+    teacher
+  end
+
+  def create_book(title, author)
+    book = Book.new(title: title, author: author)
+    @books << book
+    book
+  end
+
+  def create_rental(person, book)
+    rental = Rental.new(person: person, book: book)
+    @rentals << rental
+    rental
+  end
```

Comment:
- [ ] [REQUIRED] Good job so far! I also had the same problem with the [previous code reviewer](https://github.com/kene-creator/ruby-school-libary/pull/4/files#r1055454250). Please fix the error in `Create Rental` because `Create Rental` fails and when I do Create Person I can't even see the generated Person `ID` value also 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/209149500-408eb5ad-cba8-4199-812d-50c42481d6ea.png)